release-notes.hbs.md

```diff
@@ -33,6 +33,10 @@ This topic contains release notes for Tanzu Application Platform v1.5.
 - Introduces standardized client authentication methods to `ClientRegistration` custom resource.
   For more information, see [ClientRegistration](app-sso/crds/clientregistration.hbs.md).
 
+#### <a id='1-5-0-scc-new-features'></a> Supply Chain Choreographer
+
+- Introduces ability to configure the OOTB Basic supply chain to [output Carvel Packages](scc/carvel-package-supply-chain.hbs.md). This feature is experimental.
```

Comment:
I think 
> configure the OOTB Basic supply chain

is technically in accurate. The basic supply chains are still there as is and _along side_ them is the new one.

Perhaps we could do:

> Introduces a variation of the OOTB Basic supply chains that [output Carvel Packages](scc/carvel-package-supply-chain.hbs.md)

The above I think clearly describes **what** is new. When a reader is reviewing release notes, they may also want to know **why** it may matter to them. 

Could we append something along the lines of 
> Carvel Packages make it easy to configure per runtime environment